src/compiler/docs/test/style-docs.spec.ts

```diff
@@ -134,4 +134,17 @@ describe('style-docs', () => {
     parseStyleDocs(styleDocs, styleText);
     expect(styleDocs).toEqual([]);
   });
+
+  it('works with sass loud comments', () => {
+    const styleText = `
+      /*!
+       * @prop --max-width: Max width of the alert
+       */
+      body {
+        color: red;
+      }
+    `;
```

Comment:
Could we add a test for multiple comments with the same prop? I know it's a little weird, but was a part of the original repro case https://github.com/duhem-s/stencil-css/blob/main/src/components/my-component/my-component.css#L6